client/my-sites/checkout/checkout/checkout.jsx

```diff
@@ -341,19 +343,46 @@ export class Checkout extends React.Component {
 		if ( this.props.isNewlyCreatedSite && receipt && isEmpty( receipt.failed_purchases ) ) {
```

Comment:
> This condition will not show the upsell page to a free user who upgrades to Premium. Is this expected?
That expected for this particular test. We're not changing any of the conditions that are used to show the G Suite upsell but we need to test directly against it. There will be another test for other users that don't fit into these criteria

Reply:
> If we plan to show the upsell page only to newly created sites, then all calls to hasBloggerPlan in this context are redundant, since AFAIK we don't show the Blogger plan to newly created sites as of now (it is only seen in the Upgrade Plans page in Calypso).

That's good to know, thanks! I'll keep the check to `hasBloggerPlan` in here though for if/when we do start showing the Blogger plan to new sites.